### ISSUE TITLE
[elbow visualizer] Error in distortion score calculation

### ISSUE THREAD

**Issue by maximilian1064** [DistrictDataLabs/yellowbrick#741]
Instead of the sum of the squared distance of each sample to its closest centroid, what you are calculating here is the sum of mean distortion within each cluster. Do you intend to calculate in this way?

**Comment by maximilian1064** [comment]
https://github.com/DistrictDataLabs/yellowbrick/blob/80169ab4ac8b066abd4c5f4f8f3af34fc2c76d20/yellowbrick/cluster/elbow.py#L102

This line should be 

> distortion += distances.sum()

if you intend to follow the way described in https://bl.ocks.org/rpgove/0060ff3b656618e9136b

**Comment by bbengfort** [comment]
@maximilian1064 Thank you for reporting this, unfortunately right now we're a bit tied up with a number of PRs and open issues so we can't get to this right away. Please ping us in a week or two if we haven't followed up!

**Comment by zjpoh** [comment]
@maximilian1064 I agree with you.

According to the documentation, the distortion should be sum of squared distances, not mean. https://github.com/DistrictDataLabs/yellowbrick/blob/80169ab4ac8b066abd4c5f4f8f3af34fc2c76d20/yellowbrick/cluster/elbow.py#L49

As a simple example, consider the following four points in the same cluster.
```
X = np.array([[0, 1], [1, 0], [-1, 0], [0, -1]])
y = np.array([0, 0, 0, 0])
```
The centroid is the origin and the distortion score should be `4`. However, `distortion_score(X, y)` returns `1.0` instead.

Changing `.mean()` to `.sum()` as proposed by @maximilian1064 solves the problem.

### Code for easy reproducibility
```
import numpy as np
from yellowbrick.cluster.elbow import distortion_score
X = np.array([[0, 1], [1, 0], [-1, 0], [0, -1]])
y = np.array([0, 0, 0, 0])
distortion_score(X, y)
```

**Comment by pdamodaran** [comment]
@maximilian1064 - thanks for bringing up this issue and @zjpoh - thanks for reviewing and providing your feedback.  We will be updating the code with the suggested fix.

**Comment by maximilian1064** [comment]
@pdamodaran Thanks for the fix! I think you should also change this comment
https://github.com/DistrictDataLabs/yellowbrick/blob/80169ab4ac8b066abd4c5f4f8f3af34fc2c76d20/yellowbrick/cluster/elbow.py#L101
to `# Add the sum of square distances to the distortion`.

**Comment by bbengfort** [comment]
So I did some investigation in response to #762 -- (I added this comment there as well). Here is the original distortion score using `mean()`:

![elbow-1](https://user-images.githubusercontent.com/745966/53028298-9b4d7800-3434-11e9-8002-436e5921b841.png)

And here is the distortion score using `sum()`:

![elbow-2](https://user-images.githubusercontent.com/745966/53028311-a30d1c80-3434-11e9-93f0-f3b169e90abf.png)

The elbow is certainly less pronounced when using `sum()` and I believe this is becasue the `mean()` acts as a "normalized sum of square distances" - e.g. the larger the cluster (the more points it has) the more its distortion contributes to the the distortion score overall using the `sum` method, whereas when using the mean, each cluster contributes its distortion relative to the number of other clusters, not the number of clustered points. 

A typical scenario that happens with clustering is that one cluster becomes the "misc" or "dump' cluster, which has a high distortion and many points, other clusters have only one or two points, giving them a very low distortion score. I'm not sure how mean vs sum impacts the computation at this stage. 

In #515 we discuss updating with Elbow with an "inertia" metric - I think when we add this, we'll have the best of both words. @rebeccabilbro - does the above description make sense?
